client/src/modules/vouchers/toolkit/patient_fled/patient_fled.js

```diff
@@ -0,0 +1,193 @@
+angular.module('bhima.controllers')
+  .controller('PateintFledKitController', PateintFledKitController);
+
+PateintFledKitController.$inject = [
+  '$uibModalInstance', 'DebtorGroupService', 'NotifyService',
+  'SessionService', 'bhConstants', '$translate', 'VoucherToolkitService',
+];
+
+// Import transaction rows for a convention payment
+function PateintFledKitController(
+  Instance, DebtorGroup, Notify, Session, bhConstants, $translate, ToolKits,
+) {
+  const vm = this;
+
+  const { MAX_DECIMAL_PRECISION } = bhConstants.precision;
+
+  // global variables
+  vm.debtorGroupFilter = { is_convention : 1 };
+  vm.gridOptions = {};
+  vm.account = {};
+  vm.enterprise = Session.enterprise;
+
+  // expose to the view
+  vm.selectGroupInvoices = selectGroupInvoices;
+  vm.close = Instance.close;
+  vm.import = submit;
+
+  // Set up page elements data (debtor select data)
+  vm.onSelectDebtor = onSelectDebtor;
+  vm.onSelectAccount = onSelectAccount;
+
+  function onSelectDebtor(debtorGroup) {
+    vm.convention = debtorGroup;
+    selectGroupInvoices(vm.convention);
+  }
+
+  function onSelectAccount(account) {
+    vm.account.id = account.id;
+  }
+
+
+  // get debtor group invoices
+  function selectGroupInvoices(convention) {
+    vm.loading = true;
+
+    DebtorGroup.invoices(convention.uuid)
+      .then((invoices = []) => {
+        // total amount
+        const total = invoices.reduce(aggregate, 0);
+
+        invoices.forEach(invoice => {
+          invoice.date = new Date(invoice.date);
+        });
+
+        // sort in order
+        invoices.sort((a, b) => a.date > b.date);
+
+        // invoices
+        vm.gridOptions.data = invoices;
+
+        // make sure we are always within precision
+        vm.totalInvoices = Number.parseFloat(total.toFixed(MAX_DECIMAL_PRECISION));
+      })
+      .catch((err) => {
+        vm.hasError = true;
+        Notify.handleError(err);
+      })
+      .finally(() => {
+        vm.loading = false;
+      });
+  }
+
+  // generate transaction rows
+  function generateTransactionRows(result) {
+    const rows = [];
+
+    const cashboxAccountId = vm.account.id;
+    const conventionAccountId = result.convention.account_id;
+    const { invoices } = result;
+
+    // first, generate a cashbox row
+    const cashboxRow = ToolKits.getBlankVoucherRow();
+    cashboxRow.account_id = cashboxAccountId;
```

Comment:
This isn't a cashbox.  It is supposed to be an expense account.  See point 1 in the issue.